service/history/api/startworkflow/api.go

```diff
@@ -350,6 +350,7 @@ func (s *Starter) verifyNamespaceActive(
 	creationParams *creationParams,
 	currentWorkflowConditionFailed *persistence.CurrentWorkflowConditionFailedError,
 ) error {
+	// we are validating the version for the new workflow we are about to create, it must be running
```

Comment:
if we have create as close, what is the behavior here?

Reply:
yeah I commented it must be running because we don't have create as close today.
But even if we have that, it actually doesn't matter because there's only on transaction here, so there's only one version associated with the workflow.